.buildkite/pipeline.full.yml

```diff
@@ -0,0 +1,341 @@
+steps:
+  - label: ':docker: Build Android maze runner image'
+    key: "maze-runner"
+    timeout_in_minutes: 20
+    plugins:
+      - docker-compose#v3.3.0:
+          build: android-maze-runner
+
+  - label: ':android: NDK 12b SDK 4.4 Instrumentation tests'
```

Comment:
Would it be possible to tweak the ordering of jobs so that one mazerunner job runs after the first instrumentation test? That might help speed up feedback when Browserstack is saturated